Source/ThirdParty/ANGLE/src/common/log_utils.h

```diff
@@ -248,6 +249,23 @@ std::ostream &FmtHex(std::ostream &os, T value)
 #    define EVENT(message, ...) (void(0))
 #endif
 
+// Allow us to disable assertions at runtime by setting the ANGLE_ASSERTS_OFF environment variable
+#if defined(ANGLE_ENABLE_ASSERTS)
+static inline bool IsAssertionEnabled() {
+    static bool enabled = true;
```

Comment:
static variables are atomically initialized, so you should just skip the std::once and do
```
static bool enabled = [] { return GetEnvironmentVar("ANGLE_DISABLE_ASSERTS") != "1"; } 
```

 The code hardly needs explanation comments.